### PR TITLE
Deprecate the package

### PR DESCRIPTION
**Pull request by haaawk**

### CODE REVIEW

File: sqliteparser/README.md
Comment (sivukhin) [inline sqliteparser/README.md:1]: Why to move this out from the root directory?
Also, I think we better to write deprecation notice right in the README

**Review by sivukhin — APPROVED** [review]

File: sqliteparser/README.md
Comment (haaawk) [reply sqliteparser/README.md:1]: What do you mean by `move this out from the root directory`?

Comment (haaawk) [reply sqliteparser/README.md:1]: We can do both README and go.mod - apparently deprecation in go.mod makes tools highlight that to users

Comment (haaawk) [reply sqliteparser/README.md:1]: ok sorry. This wasn't supposed to be commited.

**Pull request merged**